arangod/Replication2/Methods.h

```diff
@@ -54,6 +58,10 @@ struct WaitForResult;
 struct ReplicatedLogMethods {
   static constexpr auto kDefaultLimit = std::size_t{10};
 
+  typedef std::variant<replication2::replicated_log::LogStatus,
+                       replication2::replicated_log::GlobalStatus>
+      GenericLogStatus;
```

Comment:
```suggestion
  using GenericLogStatus = std::variant<replication2::replicated_log::LogStatus,
                       replication2::replicated_log::GlobalStatus>;
```
we usually don't use typedef because using is more generic and we want to keep consistent.